webextensions/api/browsingData.json

```
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browsingData",
           "support": {
             "chrome": {
-              "version_added": "≤58"
+              "version_added": "≤27"
```

👉 https://github.com/chromium/chromium/commit/006236a292a8dec7b134b3db35bc1d9ea7767513 landed in [19.0.1052.0](https://github.com/chromium/chromium/releases/tag/19.0.1052.0) landed before Chrome 19.0.1084.

```suggestion
              "version_added": "19"
```